ci/run_tests.sh

```diff
@@ -43,10 +45,14 @@ do
         NUM_JOBS=2
     fi
 
-    pytest -m "$TYPE_PATTERN$PATTERN" -n $NUM_JOBS -s --strict --durations=10 --junitxml=test-data-$TYPE.xml $TEST_ARGS $COVERAGE pandas
+    PYTEST_CMD="pytest -m \"$TYPE_PATTERN$PATTERN\" -n $NUM_JOBS -s --strict --durations=10 --junitxml=test-data-$TYPE.xml $TEST_ARGS $COVERAGE pandas"
+    echo $PYTEST_CMD
+    # if no tests are found (the case of "single and slow"), pytest exits with code 5, and would make the script fail, if not for the below code
```

Comment:
if no tests are found it is an error

Reply:
we don't have tests for the builds when we filter by "slow", when we execute the "single" call here. So, all our slow builds will fail.

Reply:
ok thanks
even more explicit comments here would help then

but can be more another pass

Reply:
sure